app/scripts/pages/config.js

```diff
@@ -9,6 +9,8 @@ var Config = React.createClass({
   getInitialState: function () {
     var t = this
     t.props.ipfs.config.show(function (err, configStream) {
+      if (err) console.log(err)
+
       var config = ''
       configStream.on('data', function (chunk) {
```

Comment:
I way to make this nicer/simpler (and for any case we have a readable stream) is piping it to a writable stream and one that works really well is https://www.npmjs.com/package/bl so that:

configStream.pipe(bl(function (err, data) {})

This way, we don't have to do the chunking glue ourselves every time and we also don't forget to check for the error on the readable stream.